Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -49,27 +59,83 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   return 0;
 }
 
+// Retrieves the fuzzing target from the FUZZING_TARGET environment variable.
+// Default target is NONE if the environment variable is empty, not set, or
+// could not be interpreted. Should be kept in sync with FuzzingTarget.
+FuzzingTarget GetFuzzingTarget() {
+  NSString *fuzzing_target_env =
+      [[[NSProcessInfo processInfo] environment] objectForKey:@"FUZZING_TARGET"];
```

Comment:
Nit: in general, I would advise to convert Objective-C objects to C++ as soon as possible and proceed using C++:
```cpp
#include "Firestore/core/src/firebase/firestore/util/string_apple.h"

using firebase::firestore::util::MakeStringView;
// ...

// Get the environment variable, check for nil...
absl::string_view fuzzing_target_env = MakeStringView(fuzzing_target_env_objc);
if (fuzzing_target_env == "NONE") {
  return FuzzingTarget::kNone;
} else if (fuzzing_target_env == "SERIALIZER") {
  return FuzzingTarget::kSerializer;
}
// Log...
return FuzzingTarget::kNone;
```

In this case, feel free to leave as is, because you've written it already, but it's best to not add more new Objective-C code than necessary.